crates/telio-utils/src/repeated_actions.rs

```diff
@@ -114,6 +114,7 @@ where
             .map(|(key, interval, action)| {
                 interval
                     .map(move |instant| {
+                        dbg!(instant);
                         (key, action)
                     })
```

Comment:
I think it's enough to change line `move |instant| {` to `move |_| {`.

Reply:
Yeah, that should do the trick

Reply:
You are right, lets do that :D

Reply:
done